app/src/androidTest/java/com/duckduckgo/app/integration/AtbIntegrationTest.kt

```diff
@@ -94,7 +95,7 @@ class AtbIntegrationTest {
     }
 
     @Test
-    fun whenStatisticsAlreadyStoredThenRefreshSearchSuccessfullyUpdatesSearchRetentionAtbOnly() {
+    fun whenStatisticsAlreadyStoredThenRefreshSearchSuccessfullyUpdatesSearchRetentionAtbOnly() = runTest {
```

Comment:
Don’t think that will solve the flakiness. 

The last failure’s root cause was: 
- `StatisticsRequester$refreshSearchRetentionAtb: Search atb refresh failed with error failed to connect to duckduckgo.com/40.89.244.232 (port 443)`
- i.e., test device lost connectivity

Adding `runTest` won’t hurt anyway

Reply:
good catch. Closing this then